application/config/constants.php

```diff
@@ -82,5 +82,19 @@
     '澳门特别行政区'
 );
 
+/*
+ * Error message settings;
+ * =======================
+ * These messages are error messages returned to user input.
+ *
+ */
+$ERR_MSG = array(
+    '200' => 'OK',
+    '202' => 'User not confirmed.',
```

Comment:
We need to judge here. One option is to add period at the end of EVERY message, or remove them all.